Python/hbayesdm/base.py

```diff
@@ -423,16 +428,68 @@ def _prepare_pars(self, model_regressor: bool, inc_postpred: bool) -> List:
             pars += self.postpreds
         return pars
 
+    def _prepare_gen_init_vb(self,
+                             data_dict: Dict,
+                             n_subj: int,
+                             ) -> Union[str, Callable]:
+        """Prepare initial values for the parameters using Variational Bayesian
+        methods.
+
+        Parameters
+        ----------
+        data_dict
+            Dict holding the data to pass to Stan.
+        n_subj
+            Total number of subjects in data.
+
+        Returns
+        -------
+        gen_init : Union[str, Callable]
+            A function that returns initial values for each parameter, based on
+            the variational Bayesian method.
+        """
+        model = self._get_model_full_name()
+        sm = self._designate_stan_model(model)
```

Comment:
@JaeyeongYang 
If you call `self._designate_stan_model(model)` within `_prepare_gen_init_vb()`, then `_designate_stan_model()` will end up being called twice within the entirety of `_run()`. Because (the Stanmodel caching and) user prompting occurs in `_designate_stan_model()`, the user will be prompted the line: `'Using cached StanModel:', cache_file` a total of two times.